frida-gum/src/module.rs

```diff
@@ -23,4 +36,35 @@ impl Module {
             },
         } as *mut c_void)
     }
+
+    pub fn find_base_address(module_name: &str) -> NativePointer {
+        let module_name = CString::new(module_name).unwrap();
+
+        unsafe {
+            NativePointer(gum_sys::gum_module_find_base_address(module_name.as_ptr()) as *mut c_void)
+        }
+    }
+
+    pub fn enumerate_ranges(
+        module_name: &str,
+        prot: gum_sys::GumPageProtection,
```

Comment:
We should not be leaking `gum_sys` interfaces through the safe interface. Instead a corresponding Rust enum should be present, like `PageProtection::ReadExecute`, etc. and map to the internal one. See the Stalker enums for an example.

Reply:
Done.